src/main/webapp/app/overview/exercise-details/course-exercise-details.component.ts

```diff
@@ -66,26 +66,15 @@ export class CourseExerciseDetailsComponent implements OnInit, OnDestroy {
 
     loadExercise() {
         this.exercise = null;
-        const cachedParticipations = this.participationWebsocketService.getAllParticipationsForExercise(this.exerciseId);
```

Comment:
Caching would be a good idea, but as components can e.g. decide to only load the latest result for a participation, the result list can be incomplete.